tests/css_less/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -3276,7 +3276,7 @@ li + li {
 
 // Usage
 @import // Comment
-  "@{themes}/tidal-wave.less";
+"@{themes}/tidal-wave.less";
```

Comment:
broken

Reply:
Is expected behaviour 
```
@import "@{themes}/tidal-wave.less";
```
or the old behaviour ?

Reply:
Fixed it by checking if we're in an atRule and whether there is a prevNode. 

I'm not sure this is the right way to go about it?